tests/test_start_server.py

```diff
@@ -32,10 +31,15 @@ def test_cli_run_server_identity_path():
             encoding="utf-8",
         )
 
+        # Skip line "UserWarning: The installed version of bitsandbytes was compiled without GPU support. <...>"
```

Comment:
Can we assert that the skipped line contains this?

Then we'll be safe if smth changes, and the code will be self-descriptive (no need for the comment anymore).